frontend/src/components/common/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from 'react';
+import { Layout, Drawer } from 'antd';
+import Button from 'antd-button-color';
+import { useState } from 'react';
+import { MenuOutlined } from '@ant-design/icons';
+import logo from '../../../assets/logo.svg';
+
+const Header = Layout.Header;
+type RoutePath = 'main' | 'active' | 'drive' | 'account';
+
+export interface INavbarProps {
+  activeLabel: RoutePath;
+  setActiveLabel: React.Dispatch<React.SetStateAction<RoutePath>>;
+}
+
+const Navbar: FC<INavbarProps> = ({ ...props }) => {
+  const { activeLabel, setActiveLabel } = props;
+  const [show, setShow] = useState(false);
+
+  const buttons = [
+    { path: '/', name: 'main' },
+    { path: '/active', name: 'active' },
+    { path: '/drive', name: 'drive' },
+    { path: '/account', name: 'account' },
```

Comment:
I'd actually put the display label here, so with the uppercase already, then pass and use setActiveRoute from the parent instead of setActiveLabel

Reply:
But how can i change the passed state clicking the buttons without a setState function? Maybe i'm misunderstanding